lib/index.js

```diff
@@ -102,6 +105,15 @@ module.exports = function (gulp, options) {
     runWithDependencyProcess(runMocha, _.flattenDeep([options.unitTestFiles, options.integrationTestFiles]), cb);
   });
 
+  gulp.task('test-junit', 'Run all the tests with code coverage and output the results as JUNIT. (Specify ', ['lint-junit'], function (cb) {
```

Comment:
I see your point, and I guess you would really only use this for CI/CD anyway since it's not convenient to have test results in a file during local development.